test/e2e_node/swap_test.go

```diff
@@ -144,7 +143,7 @@ func runPodAndWaitUntilScheduled(f *framework.Framework, pod *v1.Pod) *v1.Pod {
 
 func isSwapFeatureGateEnabled() bool {
 	ginkgo.By("figuring if NodeSwap feature gate is turned on")
-	return utilfeature.DefaultFeatureGate.Enabled(features.NodeSwap)
+	return isFeatureGateEnabled(features.NodeSwap)
```

Comment:
Can't you use `utilfeature.DefaultFeatureGate.Enabled(feature)` here and remove the `isFeatureGateEnabled` function? The new comments there can be moved to this function.

Having two functions which are both just thin wrappers makes it harder to figure out what the code actually does.

Reply:
Yeah, I agree. Thanks!
Done.